src/Cake.Curl/ArgumentsExtensions.cs

```diff
@@ -77,6 +78,20 @@ internal static class ArgumentsExtensions
             {
                 arguments.Append("--retry-connrefused");
             }
+
+            if (settings.MaxTimeSeconds > 0.0)
```

Comment:
If the `MaxTime` property is a `Nullable<TimeSpan>`, we can simply say:

```csharp
if (settings.MaxTime.HasValue)
{
    ...
}
```

See [this comment](
https://github.com/ecampidoglio/Cake.Curl/pull/27/files#r268230927) for more information.